envoy_build_config/extensions_build_config.bzl

```diff
@@ -38,27 +38,27 @@ EXTENSIONS = {
     # HTTP filters
     #
 
-    "envoy.filters.http.buffer":                        "//source/extensions/filters/http/buffer:config",
+    # "envoy.filters.http.buffer":                        "//source/extensions/filters/http/buffer:config",
     "envoy.filters.http.cors":                          "//source/extensions/filters/http/cors:config",
     #"envoy.filters.http.csrf":                          "//source/extensions/filters/http/csrf:config",
     #"envoy.filters.http.dynamic_forward_proxy":         "//source/extensions/filters/http/dynamic_forward_proxy:config",
     #"envoy.filters.http.dynamo":                        "//source/extensions/filters/http/dynamo:config",
-    "envoy.filters.http.ext_authz":                     "//source/extensions/filters/http/ext_authz:config",
-    "envoy.filters.http.fault":                         "//source/extensions/filters/http/fault:config",
-    "envoy.filters.http.grpc_http1_bridge":             "//source/extensions/filters/http/grpc_http1_bridge:config",
+    #"envoy.filters.http.ext_authz":                     "//source/extensions/filters/http/ext_authz:config",
+    #"envoy.filters.http.fault":                         "//source/extensions/filters/http/fault:config",
+    #"envoy.filters.http.grpc_http1_bridge":             "//source/extensions/filters/http/grpc_http1_bridge:config",
```

Comment:
It seems that we remove almost all extensions. Since the list of our filters are so small,  will it be cleaner to just list out our wanted extensions,  instead of commenting out un-wanted ones.

Reply:
Sure, makes it much easier to read. I'll fix build errors first, then do this.

Reply:
Done